source/_integrations/google.markdown

```diff
@@ -136,49 +81,83 @@ to all the Google Calendars available to your linked account.
-    track: true
-    search: "#UnImportant"
-  - device_id: test_important
-    name: Important Stuff
-    track: true
-    search: "#Important"
-    offset: "!!"
-```
 
-From this we will end up with the binary sensors `calendar.test_unimportant` and `calendar.test_important` which will toggle themselves on/off based on events on the same calendar that match the search value set for each.
-`calendar.test_unimportant` will toggle for events whose title contain '#UnImportant'
-`calendar.test_important` will toggle for events whose title contain '#Important'. By using the offset variable an event title containing "#Important !!-10" will toggle the sensor 10 minutes before the event starts.
+{% details "Attributes" %}
+
+- **all_day**: `true`/`false` if this is an all day event. Will be `false` if there is no event found.
+- **message**: The event summary.
+- **description**: The event description.
+- **location**: The event location.
+- **start_time**: Start time of event.
+- **end_time**: End time of event.
+
+{% enddetails %}
+
+### Service `google.add_event`
+
+You can use the service `google.add_event` to create a new calendar event in a calendar. Calendar id's can be found in the file `google_calendars.yaml`. All dates and times are in your local time, the integration gets your time zone from your `configuration.yaml` file.
+
+{% details "Add Event Service details" %}
+
+<div class='note'>
+
+This will only be available if you have given Home Assistant `read-write` access in configuration options.
+
+</div>
+
+| Service data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | --------|
+| `calendar_id` | no | The id of the calendar you want. | *****@group.calendar.google.com
+| `summary` | no | Acts as the title of the event. | Bowling
+| `description` | yes | The description of the event. | Birthday bowling
+| `start_date_time` | yes | The date and time the event should start. | 2019-03-10 20:00:00
+| `end_date_time` | yes | The date and time the event should end. | 2019-03-10 23:00:00
+| `start_date` | yes | The date the whole day event should start. | 2019-03-10
+| `end_date` | yes | The date the whole day event should end. | 2019-03-11
+| `in` | yes | Days or weeks that you want to create the event in. | "days": 2
 
-<div class='note warning'>
+<div class='note'>
 
-If you use a `#` sign for `search` then wrap the whole search term in quotes.
-Otherwise everything following the hash sign would be considered a YAML comment.
+You either use `start_date_time` and `end_date_time`, or `start_date` and `end_date`, or `in`.
 
 </div>
 
+{% enddetails %}
+
+## More Configuration
+
+{% details "More Configuration" %}
+
+The integration will discover new calendars and write them into a configuration file `google_calendars.yaml` in your configuration directory that can allow for additional configuration. It is not recommended to new users to use these settings as they are not
```

Comment:
Should the sentence that discourages using this feature be a warning? Does that work within a detail?

Reply:
Looks like it works. I've split the message into two parts so the warning is a little smaller and easier to read.

Reply:
One other thing is that i've started adding some recipes to the `calendar` page https://github.com/home-assistant/home-assistant.io/pull/22846 to try to help users understand how to get some of the functionality they get with what they believe is a simple binary sensor.

Maybe it makes sense to move that here...

Reply:
I think it's better to link to a single page with examples than to add examples to multiple pages.